src/workspace-data/data-table/entity-service/EntityDeleter.js

```diff
@@ -52,7 +52,7 @@ export const EntityDeleter = ({ onDismiss, onSuccess, namespace, name, selectedE
       runningSubmissionsCount > 0 &&
         b({ style: { display: 'block', margin: '1rem 0' } }, [
           `WARNING: ${runningSubmissionsCount} workflows are currently running in this workspace. ` +
-            'Deleting the following data could cause failures if a workflow is using this data.',
+            'Deleting the following entries could cause workflows to fail if they are using the data.',
```

Comment:
nit: did you mean "entities" instead of "entries"? Honestly either way is fine :)

Reply:
I think I did mean it like "entries in this list" but I agree that "entities" is more canonical.

Reply:
There are several other places in user-facing text that refer to "entries"...

Reply:
A "table entry" contains one "entity", makes sense to me. Opting for a "when in Rome" approach based on what the dialog already uses.